packages/alea-frontend/pages/course-home/[courseId].tsx

```diff
@@ -173,6 +198,11 @@ const CourseHomePage: NextPage = () => {
             {<p>{t.practiceProblems}</p>}&nbsp;
             <Image src="/practice_problems.svg" width={35} height={35} alt="" />
           </CourseComponentLink>
+          {isInstructor && (
+            <CourseComponentLink href={`/instructor-dash/${courseId}`}>
+              <PersonIcon fontSize="large" />
```

Comment:
Add text "Instructor dashboard" and use translations (see `t.practiceProblems` above)